jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -153,7 +153,8 @@ function declare_timer_common_variables {
         local value=$(find_env "${prefix}_${var}")
         if [[ -n ${value} ]]; then
             value=$(echo ${value} | sed -e 's/\;/\\;/g')
-            eval EJB_TIMER_${var}="${value}"
+            ## https://issues.redhat.com/browse/RHPAM-3211 avoid expansion if $n is in the username/password
+            eval "EJB_TIMER_${var}=\\$value"
```

Comment:
I think it will not work, have you run all behave tests?

```
🎩 export env="TEST$n"
🎩 echo $env
TEST
🎩 echo \\$env
\TEST

```

Reply:
The original solution was =\$value tested first on openshift and it works, then I've added the second slash because this https://github.com/jboss-container-images/jboss-kie-modules/pull/398/commits/29dc7bab337663de91dd576253d86d119c110337#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cL155 introduce a \ when ; is found , anyway I'll try to run the behave tests

Reply:
@spolti splitted to avoid the double slash from the sed, behave test works, do you want a specific test only for this or is it enough change the password in other test with $0 and check the value ?